docker/puppetdb/docker-entrypoint.sh

```diff
@@ -11,10 +11,13 @@ if [ ! -d "/etc/puppetlabs/puppetdb/ssl" ] && [ "$USE_PUPPETSERVER" = true ]; th
     sleep 1
   done
   set -e
-  /opt/puppetlabs/bin/puppet config set certname "$HOSTNAME"
-  /opt/puppetlabs/bin/puppet config set server "$PUPPETSERVER_HOSTNAME"
-  /opt/puppetlabs/bin/puppet agent --verbose --onetime --no-daemonize --waitforcert 120
-  /opt/puppetlabs/server/bin/puppetdb ssl-setup -f
+  puppet config set certname "$HOSTNAME"
+  puppet config set server "$PUPPETSERVER_HOSTNAME"
+  puppet agent --verbose --onetime --no-daemonize --waitforcert 120
+  # /opt/puppetlabs/server/bin/puppetdb ssl-setup -f
+  /ssl-setup.sh -f
```

Comment:
These 3 lines are the most problematic thing here, since this means we need puppet+ruby in the image.

Sounds like we'll need to figure out a way to do this stuff differently, not sure when we'll need to do that though.